workflow-templates/node-image-build-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+name: CI Pipeline - Build Image
+
+permissions:
+  checks: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-pipeline:
+    name: "Build Pipeline"
+    uses: brandwatch/bw-workflow-actions/.github/workflows/node-build-deploy-image.yaml@production
+    with:
+      node-version: 18
```

Comment:
Comments on each of these input lines would be great. When the user clicks "New Workflow" they will get dropped into an editor where they can tweak the flow before commiting. Makes life easier if they have the docs right there in the editor with them.

Reply:
Ahhh yeah, so I'll add all the parameters and then include in the comment that it's optional

Reply:
TBH for these I would be inclined just to omit the optional ones. Will encourage new apps to use the settings we think are best. Perhaps link out to the docs for other things they can set?

Reply:
Yeah, I agree this is best.